### ISSUE TITLE
Import error in aiohttp with yarl v1.0.0

### ISSUE THREAD

**Issue by panagiks** [aio-libs/yarl#161]
Hello,

with `yarl` v1.0.0 I get the following error when doing `from aiohttp import web`:

```sh
  File "/usr/local/lib/python3.5/dist-packages/aiohttp/web.py", line 15, in <module>
    from . import (hdrs, web_exceptions, web_fileresponse, web_middlewares,
  File "/usr/local/lib/python3.5/dist-packages/aiohttp/web_middlewares.py", line 5, in <module>
    from aiohttp.web_urldispatcher import SystemRoute
  File "/usr/local/lib/python3.5/dist-packages/aiohttp/web_urldispatcher.py", line 21, in <module>
    from yarl import URL, unquote
ImportError: cannot import name 'unquote'
```

using `aiohttp-2.3.7`.

**Comment by panagiks** [comment]
Ah sorry ... I just saw in the changelog that the drop was intentional. I'll leave the issue open just for reference. Feel free to close it if you prefer :)

**Comment by foxx** [comment]
Just encountered this same problem, but it's not clear why these were dropped. Change log references https://github.com/aio-libs/yarl/pull/155 but doesn't explain why they were dropped.

**Comment by Carbaz** [comment]
This private/public "collision" is a history of a year long...

https://github.com/aio-libs/aiohttp/issues/1447

**Comment by oddjobz** [comment]
Unfortunately this has broken all my builds :( .. currently I'm trying to pin multidict and yarl to previous versions, but with no success .. not sure if I'm now hitting an issue with pipenv..
```
$ pipenv lock
Courtesy Notice: Pipenv found itself running within a virtual environment, so it will automatically use that environment, instead of creating its own for any project.
Locking [dev-packages] dependencies…
Locking [packages] dependencies…
Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
  You can use $ pipenv install --skip-lock to bypass this mechanism, then run $ pipenv graph to inspect the situation.
Could not find a version that matches multidict==3.0.0,>=2.0,>=4.0.0
Tried: 1.0.0a0, 1.0.1, 1.0.2, 1.0.3, 1.1.0b1, 1.1.0b2, 1.1.0b4, 1.1.0b5, 1.1.0b6, 1.1.0b7, 1.1.0b8, 1.1.0, 1.2.0, 1.2.1, 1.2.2, 2.0.0, 2.0.1, 2.1.0, 2.1.1, 2.1.2, 2.1.3, 2.1.4, 2.1.5, 2.1.5, 2.1.6, 2.1.6, 3.0.0, 3.0.0, 3.1.0, 3.1.0, 3.1.1, 3.1.1, 3.1.3a0, 3.1.3a0, 3.1.3, 3.1.3, 3.1.4a0, 3.1.4a1, 3.1.4a1, 3.1.4a2, 3.1.4a2, 3.1.4a3, 3.1.4a3, 3.2.0a3, 3.2.0a3, 3.2.0, 3.2.0, 3.2.1a0, 3.2.1a0, 3.3.0, 3.3.0, 3.3.1, 3.3.1, 3.3.2, 3.3.2, 4.0.0, 4.0.0
Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
  You can use $ pipenv install --skip-lock to bypass this mechanism, then run $ pipenv graph to inspect the situation.
Could not find a version that matches multidict==3.0.0,>=2.0,>=4.0.0
Tried: 1.0.0a0, 1.0.1, 1.0.2, 1.0.3, 1.1.0b1, 1.1.0b2, 1.1.0b4, 1.1.0b5, 1.1.0b6, 1.1.0b7, 1.1.0b8, 1.1.0, 1.2.0, 1.2.1, 1.2.2, 2.0.0, 2.0.1, 2.1.0, 2.1.1, 2.1.2, 2.1.3, 2.1.4, 2.1.5, 2.1.5, 2.1.6, 2.1.6, 3.0.0, 3.0.0, 3.1.0, 3.1.0, 3.1.1, 3.1.1, 3.1.3a0, 3.1.3a0, 3.1.3, 3.1.3, 3.1.4a0, 3.1.4a1, 3.1.4a1, 3.1.4a2, 3.1.4a2, 3.1.4a3, 3.1.4a3, 3.2.0a3, 3.2.0a3, 3.2.0, 3.2.0, 3.2.1a0, 3.2.1a0, 3.3.0, 3.3.0, 3.3.1, 3.3.1, 3.3.2, 3.3.2, 4.0.0, 4.0.0
```
I have multidict pinned to 3.0.0 and yarl to 0.11.0 ... any ideas on how to get around this would be very welcome ... !

**Comment by oddjobz** [comment]
Ok, scratch that, managed to fix it by pinning aiohttp to 2.3.8 and unpinning yarl and multidict ...

**Comment by asvetlov** [comment]
aiohttp is fixed
